docs/site/Configuring-datasource.md

```diff
@@ -0,0 +1,23 @@
+---
+lang: en
+title: 'Configuring DataSource'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Connector, Datasource
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Configuring-datasource.html
+---
+
+## Overview
```

Comment:
Please add a short note to explain readers that this is a placeholder page and add a link to the GH issue that's tracking the task of writing the actual content. See https://github.com/strongloop/loopback-next/blob/master/docs/site/migration/clients.md for inspiration.

```
{% include note.html content="
This is a placeholder page, the task of adding content is tracked by the following GitHub issue: 
[loopback-next#3954](https://github.com/strongloop/loopback-next/issues/3954)
" %}
```